src/Robo/Commands/Sync/RefreshCommand.php

```diff
@@ -56,12 +56,7 @@ protected function refreshMultisite($multisite_name) {
    * @command sync:refresh
```

Comment:
Funny, I had this exact same idea previously and even began to implement it. I was thinking that each top level command, e.g. setup, tests, sync,  could be structured this way.

We should probably make an alias for this command that is simply `sync`,  so that it corresponds with having configuration that lives under the key `sync.commands`.